nengo_loihi/loihi_cx.py

```diff
@@ -496,6 +496,7 @@ def __init__(self, model, seed=None):
 
         self._probe_filters = {}
         self._probe_filter_pos = {}
+        self.closed = False
```

Comment:
Safest to put this line first, before `self.build`, in case we ever make this a context manager in the future (we had some instances in `nengo.Simulator` where we would be looking at `self.closed` in the `__exit__`, which gave an attribute error).